src/patternfly/demos/Toolbar/examples/Toolbar.md

```diff
@@ -418,7 +418,7 @@ import './Toolbar.css'
             {{/menu}}
           {{/panel}}
         {{/toolbar-item}}
-        {{#> toolbar-item toolbar-item--modifier=(concat 'pf-m-chip-group ' toolbar-item-chip-group--modifier)}}
+        {{#> toolbar-item toolbar-item--modifier=(concat 'pf-m-label-group ' toolbar-item-label-group--modifier)}}
```

Comment:
Doesn't look like there are any styles for a label group modifier on a toolbar item.